test/e2e/validation_test.go

```diff
@@ -464,6 +465,68 @@ func testExpectedServicesRunning(t *testing.T) {
 	}
 }
 
+// testServicesConfigMap tests 2 aspects of expected functionality for the services ConfigMap
+// 1. It exists on operator startup and 2. It is recreated when deleted
+func testServicesConfigMap(t *testing.T) {
+	tc, err := NewTestContext()
+	require.NoError(t, err)
+
+	operatorVersion, err := getWMCOVersion()
+	require.NoError(t, err)
+	// There is no easy way to populate ldflags when running e2e tests so that version.Get() will return
+	// anything other than "" without having to maintain the WMCO version in another location
+	servicesConfigMapName := servicescm.ServicesConfigMapName + operatorVersion
```

Comment:
Do you need the `-` between `servicescm.ServicesConfigMapName` and `operatorVersion`?